config/rbac/role_binding.yaml

```diff
@@ -19,7 +19,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-baremetal-operator
+  name: cluster-admin
```

Comment:
Do we need 2 cluster roles cluster-baremetal-operator and cluster-admin here?

Reply:
I don't know to be honest, what I've don't appears to work but mightn't be correct

Reply:
i think it would be better if possible to keep the existing name

Reply:
I've added configmaps to the cluster-baremetal-operator ClusterRole which I think is what I should have been doing in the first place.